app/models/state_file1099_r.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: state_file1099_rs
+#
+#  id                                 :bigint           not null, primary key
+#  capital_gain_amount                :integer
+#  designated_roth_account_first_year :integer
+#  distribution_code                  :string
+#  federal_income_tax_withheld_amount :integer
+#  gross_distribution_amount          :integer
+#  intake_type                        :string           not null
+#  payer_address_line1                :string
+#  payer_address_line2                :string
+#  payer_city_name                    :string
+#  payer_identification_number        :string
+#  payer_name                         :string
+#  payer_name_control                 :string
+#  payer_state_code                   :string
+#  payer_state_identification_number  :string
+#  payer_zip                          :string
+#  phone_number                       :string
+#  recipient_name                     :string
+#  recipient_ssn                      :string
+#  standard                           :boolean
+#  state_code                         :string
+#  state_distribution_amount          :integer
+#  state_specific_followup_type       :string
+#  state_tax_withheld_amount          :integer
```

Comment:
With our new monetary input fields, at a minimum `state_distribution_amount` and `state_tax_withheld_amount` will need to be decimals. 

A case could be made that the other monetary amounts also should be decimals, but if they aren't being edited it's not a strict requirement. (though I would lean towards all decimals anyway)